extended/src/main/java/apoc/ml/OpenAI.java

```diff
@@ -220,4 +227,30 @@ public Stream<MapResult> chatCompletion(@Name("messages") List<Map<String, Objec
       } ] }
     */
     }
+
+    static boolean checkNullInput(Object input, boolean shuldFail){
+        return checkInput(input, shuldFail, () -> Objects.isNull(input), ERROR_NULL_INPUT);
+    }
+
+    static boolean checkEmptyInput(Collection<?> input, boolean shuldFail){
+        return checkInput(input, shuldFail, () -> input.isEmpty(), ERROR_EMPTY_OR_BLANK_INPUT);
+    }
+
+    static boolean checkBlankInput(String input, boolean shuldFail){
+        return checkInput(input, shuldFail, () -> StringUtils.isBlank(input), ERROR_EMPTY_OR_BLANK_INPUT);
+    }
+
+    private static boolean checkInput(
+            Object input, boolean shuldFail,
+            Supplier<Boolean> checkFunction,
+            String exceptioMessage
+    ){
+        boolean check = true;
+        if (checkFunction.get()) {
+            if(shuldFail) throw new RuntimeException(exceptioMessage);
+            else check = false;
+        }
+        return check;
```

Comment:
dovrebbe funzionare anche così

```suggestion
        if (checkFunction.get()) {
            if(shuldFail) throw new RuntimeException(exceptionMessage);
            return false;
        }
        return true;
```